### ISSUE TITLE
If json files are added to folder then the wrong json is loaded.

### ISSUE THREAD

**Issue by 38github** [keyth72/SmartGuitarPedal#22]
This is in Reaper (Linux). Does SmartPedal not lock for the name of the json but instead what position among files like i.e. "the seventh file"?

**Comment by GuitarML** [comment]
Yes, this is a problem that has been fixed for Proteus, so I just need to do the same update to SmartPedal to reference the file name instead of the index number. I’ll include this in the next release.
